app/controllers/OtherInformationController.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions._
+import forms.OtherInformationFormProvider
+import models.Mode
+import navigation.Navigator
+import pages.unsatisfactory._
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import utils.JsonOptionFormatter
+import views.html.OtherInformationView
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class OtherInformationController @Inject()(
+                                            override val messagesApi: MessagesApi,
+                                            override val sessionRepository: SessionRepository,
+                                            override val navigator: Navigator,
+                                            override val auth: AuthAction,
+                                            override val withMovement: MovementAction,
+                                            override val getData: DataRetrievalAction,
+                                            override val requireData: DataRequiredAction,
+                                            formProvider: OtherInformationFormProvider,
+                                            val controllerComponents: MessagesControllerComponents,
+                                            view: OtherInformationView
+                                          ) extends BaseNavigationController with AuthActionHelper with JsonOptionFormatter {
```

Comment:
Won't the JsonOption handling, as the form can just be of type `Form[String]`
```suggestion
                                          ) extends BaseNavigationController with AuthActionHelper {
```

Reply:
@MJCallahanPage I'm not saying this is a copy/pasting oversight, but I can't say I fully wrote this from scratch... 👀 will change